### PR TITLE
Add tooltip to upload button in EditVideoForm

### LINKED ISSUE #1275: Add tooltip to upload button in EditVideoForm
image
Also, we should update button text

### PR DESCRIPTION
**Pull request by rafalpawlow**
Closes #1275

### CODE REVIEW

**Comment by drillprop** [comment]
made some improvements based on updated design: https://www.figma.com/file/Pf31tuYpozYmpq163U2ho8/Web-Components?node-id=798%3A90836
I think we could ignore for now checking if all required inputs are filled and deal with it in another PR.

File: src/shared/components/ActionBar/ActionBar.style.ts [hunk 1]
```diff
@@ -78,7 +78,12 @@ export const DetailsContainer = styled.div`
   margin-left: auto;
   display: flex;
   align-items: center;
-  padding: 0 ${sizes(5)};
+  padding: 13px ${sizes(4)};
+  transition: background-color ${transitions.timings.sharp} ${transitions.easing};
+
+  :hover {
+    background-color: ${colors.transparentPrimary[18]};
+  }
```
Comment (kdembler) [inline src/shared/components/ActionBar/ActionBar.style.ts:87]: Now that this element has a hover state, I think we should also make the text non-selectable so you don't get a text selection cursor when hovering over text

File: src/shared/components/ActionBar/ActionBar.style.ts [hunk 2]
```diff
@@ -78,7 +78,12 @@ export const DetailsContainer = styled.div`
   margin-left: auto;
   display: flex;
   align-items: center;
-  padding: 0 ${sizes(5)};
+  padding: 13px ${sizes(4)};
```
Comment (kdembler) [inline src/shared/components/ActionBar/ActionBar.style.ts]: I think we actually don't need any vertical padding as it should fill the entire available height anyway, right? For horizontal padding, design uses 20px

**Review by kdembler — CHANGES_REQUESTED** [review]

File: src/shared/components/Tooltip/Tooltip.tsx [hunk 3]
```diff
@@ -30,7 +30,7 @@ export const Tooltip: React.FC<TooltipProps> = ({
   arrowDisabled,
   className,
 }) => {
-  const [isVisible, setIsVisible] = useState(false)
+  const [isVisible, setIsVisible] = useState(true)
```
Comment (kdembler) [inline src/shared/components/Tooltip/Tooltip.tsx]: Where is this change coming from?

Comment (drillprop) [inline src/shared/components/Tooltip/Tooltip.tsx]:
```suggestion
  const [isVisible, setIsVisible] = useState(false)
```

Comment (drillprop) [reply src/shared/components/Tooltip/Tooltip.tsx]: damn it 😅 . I should more carefully check what I'm committing.

**Review by kdembler — APPROVED** [review]

**Pull request merged**